src/devices/Ads1115/Ads1115.cs

```diff
@@ -61,38 +79,136 @@ public DataRate DataRate
             }
         }
 
+        /// <summary>
+        /// ADS1115 operation mode.
+        /// When set to <see cref="DeviceMode.Continuous"/> the chip constantly measures the input and the values can be read directly.
+        /// If set to <see cref="DeviceMode.PowerDown"/> the chip enters idle mode after each conversion and
+        /// a new value will be requested each time a read request is performed. This is the recommended setting when constantly
+        /// swapping between input channels, because a change of the channel requires a new conversion anyway.
+        /// </summary>
+        public DeviceMode DeviceMode
+        {
+            get
+            {
+                return _deviceMode;
+            }
+            set
+            {
+                _deviceMode = value;
+                SetConfig();
+            }
+        }
+
+        /// <summary>
+        /// Comparator mode.
+        /// Only relevant if the comparator trigger event is set up and is changed by <see cref="EnableComparator(short, short, ComparatorMode, ComparatorQueue, GpioController, int)"/>.
+        /// </summary>
+        public ComparatorMode ComparatorMode
```

Comment:
This could be just `public ComparatorMode ComparatorMode => _comparatorMode` or even better `public ComparatorMode ComparatorMode { get; }` and remove the explicit field